CHANGELOG.md

```diff
@@ -56,15 +56,13 @@ Read https://github.com/appium/appium-safari-driver for more details.
 
 ### Mac2
```

Comment:
for Safari driver it makes sense to mention that it does not support webviews automation, while geckodriver does